coss/club/templates/pages/registration.html

```diff
@@ -0,0 +1,170 @@
+{% extends "pages/clubs.html" %}
+{% load i18n staticfiles %}
+
+{% block meta_title %}{% trans "Club Registration" %}{% endblock %}
+{% block title %}{% trans "Club Registration" %}{% endblock %}
+
+{% block nav-class %}nav-invert{% endblock %}
+
+{% block jumbotron %}{% endblock %}
+
+{% block main %}
+  <div class="breadcrumbs">
+    <a href="/clubs/">
+      <img class="ml-2" src="/static/img/left-arrow-small-black.svg" alt="back">
+      Open Source Clubs
+    </a>
+  </div>
+
+  <h3 class="highlight mt-4">Club Registration</h3>
+
+  <div class="row club-registration mt-4">
```

Comment:
You can definitely override style and class attributes in Django forms. We are doing this in all of our projects. Anyway, let's not block the PR because of this, although it shouldn't be this way.